frontend/src/components/input/ListInput.vue

```diff
@@ -0,0 +1,145 @@
+<script lang="ts" setup>
+import { ref, computed, watch, nextTick, onMounted } from "vue"
+import Checkbox from "@/components/input/CheckboxInput.vue"
+import InputField, { LabelPosition } from "@/components/input/InputField.vue"
+import TextButton from "@/components/input/TextButton.vue"
+
+const props = defineProps<{
+  label: string
+  modelValue: string[]
+}>()
+
+const emit = defineEmits<{
+  "update:modelValue": [value: string[]]
+}>()
+
+const editMode = ref(true)
+const sortAlphabetically = ref(false)
+const localModelValue = ref(props.modelValue ?? [])
+
+const textAreaInput = computed({
```

Comment:
If we split the EditComponent into its own value, I would rather pass the initial value as a prop and then keep the state locally in the component and emit it once the submit button is clicked. Would be happy to discuss this, as the consequences are not clear to me.
While trying the feature, I noticed that the server values override the local values. (But that might be a local problem.)

Reply:
Good catch, should be fixed with the edit component now :)